packages/common/src/ui-common/localization/mock-localizer.ts

```diff
@@ -0,0 +1,16 @@
+import { Localizer } from "./localizer";
+
+export class MockLocalizer implements Localizer {
```

Comment:
Let's call this something like StandardLocalizer instead of MockLocalizer. This isn't really a mock implementation